device/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.Azure.Amqp.Framing;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Shared;
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace device
+{
+    class Program
+    {
+        static bool sendModelAtConnection = true;
+        static bool sendModelAsProperties = true;
+        static string cs = System.Environment.GetEnvironmentVariable("DEVICE_CS");
+        static string model = File.ReadAllText(@"..\..\..\deviceModel.json");
```

Comment:
I think if this is going to be a prototype of how a device would manage a dynamic model, we shouldn't just load it from a file. How is a device going to dynamically build a model?

What happens if it changes after a connection has been established? Is that legal, or must a device disconnect and reconnect?

Reply:
SDD enables dynamic model, but it's not the same thing. We see this as the foundation to enable dynamic models. Yes the devices will need to reconnect to update the model.

Reply:
So they just don't want to publish the static model to a repo somewhere, eh?

You see, this is one example of why your code could use comments.